desktop-src/SetupApi/run-and-runonce-registry-keys.md

```diff
@@ -20,6 +20,10 @@ The Windows registry includes the following four `Run` and `RunOnce` keys:
 -   **HKEY\_CURRENT\_USER\\Software\\Microsoft\\Windows\\CurrentVersion\\Run**
 -   **HKEY\_CURRENT\_USER\\Software\\Microsoft\\Windows\\CurrentVersion\\RunOnce**
 
+> [!IMPORTANT]
+> **HKEY\_LOCAL\_MACHINE\\Software\\Microsoft\\Windows\\CurrentVersion\\RunOnce** only executes when members of the Administrators group log on after the reboot.
+> Additional details can be found in the articles [RunOnce Registry Key](https://learn.microsoft.com/en-us/windows-hardware/drivers/install/runonce-registry-key) and [Developing Applications that Run at Logon](https://learn.microsoft.com/en-us/previous-versions/bb325654(v=msdn.10)#developing-a-multi-part-setup-application)
```

Comment:
Can you update these links to be site-relative? I.e. Just omit `https://learn.microsoft.com/en-us` so that the URLs begin with a forward slash `/windows-hardware` and `/previous-versions`. This allows the links to resolve in the offline documentation in Visual Studio.

Reply:
I've updated the PR to use site-relative links (and I added an additional link)